helm/hpcc/templates/eclagent.yaml

```diff
@@ -66,6 +66,7 @@ data:
     global:
 {{ include "hpcc.generateGlobalConfigMap" $ | indent 6 }}
 {{- if not .useChildProcesses }} 
+{{- $jobNamePrefix := printf "%s-%s-" .name $apptype }}
```

Comment:
as discussed here and offline, the name of the pod itself should change, so it includes the target (.name) too, so it's clear that these dynamic jobs belong.. (when pods list viewed in k8s).
i.e. change line 74 to include .name too.